.travis.yml

```diff
@@ -9,13 +9,14 @@ php:
 cache:
   directories:
     - $HOME/.composer/cache
+    - jackalope-standalone-$JACKRABBIT_VERSION.jar
 
 env:
   global:
     - JACKRABBIT_VERSION=2.12.0
 
 before_script:
-  - wget http://archive.apache.org/dist/jackrabbit/$JACKRABBIT_VERSION/jackrabbit-standalone-$JACKRABBIT_VERSION.jar
+  - if [ ! -f jackalope-standalone-$JACKRABBIT_VERSION ]; then wget http://archive.apache.org/dist/jackrabbit/$JACKRABBIT_VERSION/jackrabbit-standalone-$JACKRABBIT_VERSION.jar; fi
```

Comment:
I think that would be sensible.

On Thu, Apr 07, 2016 at 01:28:17AM -0700, Daniel Rotter wrote:
>    In [1].travis.yml:
> 
>  >
>  >  env:
>  >    global:
>  >      - JACKRABBIT_VERSION=2.12.0
>  >
>  >  before_script:
>  > -  - wget http://archive.apache.org/dist/jackrabbit/$JACKRABBIT_VERSION/jackrabbit-standalone-$JACKRABBIT_VERSION.jar
>  > +  - if [ ! -f jackalope-standalone-$JACKRABBIT_VERSION ]; then wget http://archive.apache.org/dist/jackrabbit/$JACKRABBIT_VERSION/jackrabbit-standalone-$JACKRABBIT_VERSION.jar; fi
> 
>    Shouldn't the if also contain the .jar ending?
> 
>    —
>    You are receiving this because you modified the open/close state.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2189#discussion_r58835952
>    2. https://github.com/sulu/sulu/pull/2189/files/16981decaca3f54732153c73621beb60df0b28d6#r58835952
>    3. https://github.com/sulu/sulu/pull/2189/files/16981decaca3f54732153c73621beb60df0b28d6#r58835952